core/arch/arm/plat-imx/drivers/tzc380.c

```diff
@@ -50,5 +50,7 @@ void imx_configure_tzasc(void)
 		region = tzc_auto_configure(CFG_SHMEM_START, CFG_SHMEM_SIZE,
 			     TZC_ATTR_SP_ALL, region);
 		DMSG("Action register: %xl", tzc_get_action());
+		if (tzc_regions_lockdown() < 0)
+			EMSG("Region lockdown failed!");
```

Comment:
Suggest to panic or report the error.
Note that next commit in the series reports the error in the `driver_init` handler which... will do nothing more than printing a debug trace.